molgenis-api-permissions/src/main/java/org/molgenis/api/permissions/model/response/ObjectPermissionResponse.java

```diff
@@ -0,0 +1,24 @@
+package org.molgenis.api.permissions.model.response;
+
+import com.google.auto.value.AutoValue;
+import java.util.Set;
+import javax.annotation.Nullable;
+import org.molgenis.util.AutoGson;
+
+@AutoValue
+@AutoGson(autoValueClass = AutoValue_ObjectPermissionResponse.class)
+@SuppressWarnings(
+    "squid:S1610") // Abstract classes without fields should be converted to interfaces
+public abstract class ObjectPermissionResponse {
+  public abstract String getId();
+
+  @Nullable
+  public abstract String getLabel();
```

Comment:
null label might be tricky to handle for the client?

Reply:
removed this annotation, this is not marked outdated due to the fact that the commented line in unchanged